src/main/java/com/lunark/lunark/service/PropertyServiceImpl.java

```diff
@@ -72,4 +74,23 @@ public Property update(Property property) {
     public void delete(Long id) {
 
     }
+
+    @Override
+    public Double getAverageGrade(Long id) {
+        Optional<Property> property = this.find(id);
+        if (property.isEmpty()){
+            return -1.0;
```

Comment:
Same as with the same method in AccountService.
```suggestion
            return null;
```